### PR TITLE
Bugfix - Variant status new

### PR DESCRIPTION
**Pull request by arjun-m06**
Variant Status in catalog product component

### CODE REVIEW

**Review by joneskebhart9787 — DISMISSED** [review]

**Review by visak-kumar — DISMISSED** [review]

File: components/NewSupplementsModal/CatalogProducts.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Button, Dropdown } from "@/components/FormComponents";
 import translations from "@/components/translations";
 import { ToasterButton } from "../Protocols/ToasterButton";
```
Comment (joneskebhart9787) [inline components/NewSupplementsModal/CatalogProducts.tsx:4]: Please remove unused imports

File: components/NewSupplementsModal/CatalogProducts.tsx [hunk 2]
```diff
@@ -55,10 +55,22 @@ export function CatalogProducts({
...
   const { name } = productType(variantStatus);
   useEffect(() => {
-    if (variantStatus != undefined && name == VARIANT_STATUSES.DISCONTINUED) {
+    if (variantStatus != undefined && name != EMPTY) {
       setVariantType(true);
+      if (name == VARIANT_STATUSES.DISCONTINUED) {
+        setVarient(VARIANT_STATUSES.DISCONTINUED)
+      } else if (name == VARIANT_STATUSES.BACK_ORDER) {
+        setVarient(VARIANT_STATUSES.BACK_ORDER)
+      } else if (name == VARIANT_STATUSES.SPECIAL_ORDER) {
```
Comment (joneskebhart9787) [inline components/NewSupplementsModal/CatalogProducts.tsx:67]: Please change to switch case

File: components/NewSupplementsModal/CatalogProducts.tsx (hunk 1 above)
Comment (joneskebhart9787) [reply components/NewSupplementsModal/CatalogProducts.tsx:4]: Please remove all unused variables and probs from this file

File: components/RecommendationPage/ProductList.tsx [hunk 3]
```diff
@@ -1,30 +1,41 @@
...
+  variantStatus?: any
 }
 
 interface ProductListProps {
   options: ProductListOptions[]
 }
 export function ProductList({ options }: ProductListProps) {
-  const[more, setMore] = useState(false);
+  const [more, setMore] = useState(false);
+
```
Comment (joneskebhart9787) [inline components/RecommendationPage/ProductList.tsx:22]: Please remove line space

File: components/RecommendationPage/ReviewFulPlanProductList.tsx [hunk 4]
```diff
@@ -29,29 +29,40 @@ export interface ReviewFullPlanProductList {
...
-        }
-        else{
-            setVariantType(false)
+        if (variantStatus != undefined && name != EMPTY) {
+            setVariantType(true);
+            if (name == VARIANT_STATUSES.DISCONTINUED) {
+                setVarient(VARIANT_STATUSES.DISCONTINUED)
+            } else if (name == VARIANT_STATUSES.BACK_ORDER) {
+                setVarient(VARIANT_STATUSES.BACK_ORDER)
+            } else if (name == VARIANT_STATUSES.SPECIAL_ORDER) {
```
Comment (joneskebhart9787) [inline components/RecommendationPage/ReviewFulPlanProductList.tsx:57]: Please change it to switch case. I am seeing same codes in two places. Please twite a common function

File: components/NewSupplementsModal/CatalogProducts.tsx (hunk 1 above)
Comment (arjun-m06) [reply components/NewSupplementsModal/CatalogProducts.tsx:4]: removed

File: components/NewSupplementsModal/CatalogProducts.tsx (hunk 2 above)
Comment (arjun-m06) [reply components/NewSupplementsModal/CatalogProducts.tsx:67]: changed

File: components/RecommendationPage/ProductList.tsx (hunk 3 above)
Comment (arjun-m06) [reply components/RecommendationPage/ProductList.tsx:22]: removed

File: components/RecommendationPage/ReviewFulPlanProductList.tsx (hunk 4 above)
Comment (arjun-m06) [reply components/RecommendationPage/ReviewFulPlanProductList.tsx:57]: done

File: tailwind.config.js [hunk 5]
```diff
@@ -74,6 +74,7 @@ module.exports = {
         bold: 800,
       },
       fontSize: {
+        zxsm: ["0.6rem", "0.7rem"],
         xxxs: ["1.0rem", "1.4rem"],
         xxxsm: ["0.8rem", "0.9rem"],
```
Comment (visak-kumar) [inline tailwind.config.js:79]: Why we are using different naming?

**Review by visak-kumar — APPROVED** [review]

**Pull request merged**